frontend/src/views/GMembers.vue

```diff
@@ -772,11 +774,11 @@ function sortItems (items, sortByArr, secondSortCriteria) {
 }
 
 function setSelectedHeaderUserAccount (header) {
-  userAccountSelectedColumns.value[header.key] = !header.selected
+  userSelectedColumns.value[header.key] = !header.selected
```

Comment:
Column selection on the Members page for both tables does not work anymore

Reply:
✔️